inspec/a2-api-integration/controls/iam/admin_token.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../../../constants'
+
+# encoding: utf-8
+# copyright: 2017, Chef Software, Inc.
+# license: All rights reserved
+title 'iam introspection'
+
+control 'iam-admin-token-1' do
+  title 'IAM admin token'
+  desc 'test IAM admin token'
+
+  # only run this if we've been handed an admin token to use here
+  only_if { ENV['INSPEC_UPGRADE_IAM_V2_3_ADMIN_TOKEN'] }
```

Comment:
What does the "3" represent?

Reply:
I'm not sure. I assume that's a configured name so I didn't change it

Reply:
Super strange name for sure:

https://github.com/chef/automate/blob/master/integration/tests/iam_v2_with_v1_policies.sh#L50